OpenRA.Game/Support/Log.cs

```diff
@@ -90,7 +90,10 @@ public static void Write(string channel, string format, params object[] args)
 			if (writer == null)
 				return;
 
-			writer.WriteLine(format, args);
+			string strToWrite = string.Format(format, args);
```

Comment:
Please use `var` instead of `string`. We usually don't abbreviate variable names too, so either `stringToWrite` or just `toWrite` would be better.

Reply:
Done.